libs/ui-toolkit/modals/src/lib/modal/simple-modal/simple-modal.stories.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, Inject, Input, OnChanges, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialog, MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { PipesModule } from '@gorilainvest/ui-toolkit/pipes';
+import { boolean, text, withKnobs, object } from '@storybook/addon-knobs';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { timer } from 'rxjs';
+
+import { SimpleModalData, SocialMediaArray } from './simple-modal.data';
+import { SimpleModalComponent } from './simple-modal.component';
+import { ModalModule } from '../modal.module';
+
+const socialMediaMock = [
+  {
+    img: '/assets/img/facebook.svg',
+    URL: 'https://www.facebook.com/GorilaInvest/',
+    id: 'Facebook'
+  },
+  {
+    img: '/assets/img/instagram.svg',
+    URL: 'https://www.instagram.com/gorilainvest',
+    id: 'Instagram'
+  },
+  {
+    img: '/assets/img/twitter.svg',
+    URL: 'https://twitter.com/gorilainvest',
+    id: 'Twitter'
+  },
+  {
+    img: '/assets/img/linked-in.svg',
+    URL: 'https://www.linkedin.com/company/gorila',
+    id: 'Linked-in'
+  }
+];
```

Comment:
Instead of repeating the code here and in the .ts file of the component, you could define this as a const exported in simple-modal.data.ts and then import it in both files. Use a name that indicates this is the default value, such as 'defaultSocialMediaIcons'